march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -372,6 +380,31 @@ void MarchHardwareInterface::updatePowerNet()
   }
 }
 
+void MarchHardwareInterface::updateAfterLimitJointCommand()
+{
+  return;
+  if (!after_limit_joint_command_pub_->trylock())
+  {
+    return;
+  }
+
+  // Clear msg of AfterLimitJointCommand
+  after_limit_joint_command_pub_->msg_.name.clear();
+  after_limit_joint_command_pub_->msg_.position_command.clear();
+  after_limit_joint_command_pub_->msg_.effort_command.clear();
+
+  for (int i = 0; i < num_joints_; i++)
+  {
+    march4cpp::Joint singleJoint = marchRobot.getJoint(joint_names_[i]);
```

Comment:
I think `joint` is sufficient, more consistent with other code and I don't understand what `single` adds here.